stable/cratedb/templates/internal-service.yaml

```diff
@@ -0,0 +1,24 @@
+# CrateDB internal service
+# This is the service that each CrateDB node be able to discover and
+# communicate with the other nodes in the cluster.
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{  .Values.internalService.name }}
+  namespace: {{ .Values.namespace }}
+  labels:
+    app.kubernetes.io/name: {{ template "cratedb.name" . }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+    helm.sh/chart: {{ include "cratedb.chart" . }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/version: {{ .Chart.Version }}
+{{- if .Values.labels }}
+{{ toYaml .Values.labels | indent 4 }}
+{{- end }}
+spec:
+  type: ClusterIP
+  ports:
+    - port: {{ .Values.internalService.port }}
+      name: {{ .Values.internalService.portName }}
+  selector:
+    app: {{ template "cratedb.name" . }}
```

Comment:
this label name is not correct, use `app.kubernetes.io/name` instead of `app` 
please see: https://github.com/helm/charts/blob/master/REVIEW_GUIDELINES.md#service-selectors